### ISSUE TITLE
OCRed text cut too narrow - how to correct or report

### ISSUE THREAD

**Issue by mirjam-amsterdam** [urieli/jochre#43]
in this snippet an nun went missing. (or instead of nor) most probably the text-width of the scanned book was set too narrow.
Should I just add the nun, or should I report such cases to github?

https://archive.org/stream/nybc203972#page/n176/mode/1up

![too narrow letter missing](https://user-images.githubusercontent.com/44893880/66113979-2a775c80-e5ce-11e9-935b-74e726e6dc20.png)

**Comment by urieli** [comment]
No, do not correct such cases, because the text will no longer correspond to the image within the word boundaries.
It's better to report this on Github. Hopefully such issues will be vastly reduced in Jochre 3, better segmentation being our first objective.
